util/getGuildInfo.js

```
@@ -1,9 +1,16 @@
 module.exports = function(client){
 	let guildCount = 0;
 	let totalMembers = 0;
+	let uniqueMembers = new Map();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'uniqueMembers' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)